storage/index.js

```diff
@@ -1 +1,15 @@
 // JavaScript source code
+var server = require('./app.js');
+var storageModule;
+var config = server.config;
+var client;
+
+// Use the following to require different types of DB
+if (config.db.type =='mongo'){
+    storageModule = require('./mongo/index.js');
+    client = new storageModule(config);
```

Comment:
No need to use here  you can use storageModule Directly. Apart from that if else wont work because in case of else condition client wont be able available. anyways client variable is not needed at all storageModule can work.